diff --git a/platform/Kconfig b/platform/Kconfig
index 5f6e78c..4fe7b96 100644
--- a/platform/Kconfig
+++ b/platform/Kconfig
@@ -3,6 +3,7 @@ config PLATFORM
     default "hi3516dv300"     if PLATFORM_HI3516DV300
     default "hi3518ev300"     if PLATFORM_HI3518EV300
     default "virt"            if PLATFORM_QEMU_ARM_VIRT_CA7
+    default "v3s"             if PLATFORM_ALLWINNER_V3S
 
 config PRODUCT_NAME
     string "unknown"
@@ -11,12 +12,14 @@ config PRODUCT_NAME
     default "hispark_taurus" if PRODUCT_HISPARK_TAURUS
     default "hispark_aries" if PRODUCT_HISPARK_ARIES
     default "arm_virt" if PRODUCT_QEMU_ARM
+    default "v3s" if PRODUCT_ALLWINNER_V3S
 
 config DEVICE_COMPANY
     string "unknown"
     default "hisilicon" if PLATFORM_HI3516DV300
     default "hisilicon" if PLATFORM_HI3518EV300
     default "qemu" if PLATFORM_QEMU_ARM_VIRT_CA7
+    default "allwinner" if PLATFORM_ALLWINNER_V3S
 
 choice
     prompt "Chip"
@@ -42,6 +45,10 @@ config PLATFORM_QEMU_ARM_VIRT_CA7
     help
         QEMU ARM Virtual Platform using Cortex-A7 CPU.
 
+config PLATFORM_ALLWINNER_V3S
+    bool "v3s"
+    select ARCH_CORTEX_A7
+
 endchoice
 
 choice
@@ -61,6 +68,10 @@ config PRODUCT_HISPARK_ARIES
 
 config PRODUCT_QEMU_ARM
     bool "arm_virt" if PLATFORM_QEMU_ARM_VIRT_CA7
+
+config PRODUCT_ALLWINNER_V3S
+    bool "v3s_product" if PLATFORM_ALLWINNER_V3S
+
 endchoice
 
 config BOARD_CONFIG_PATH
